src/proto_tcp.c

```diff
@@ -692,7 +692,13 @@ int tcp_bind_listener(struct listener *listener, char *errmsg, int errlen)
 		int defaultmss;
 		socklen_t len = sizeof(tmpmaxseg);
 
-		if (listener->rx.addr.ss_family == AF_INET)
+		int mptcp = listener->rx.proto->sock_prot == IPPROTO_MPTCP;
```

Comment:
(in their code style, variables should be declared at the top of a block, without new lines in between)